### DISCUSSION TITLE
Hello, I can't access the site from the pool of IP 93.170.44.0/22 (Ukraine, Dnipro, SOYUZTELECOM)

### DISCUSSION THREAD

**Discussion by alexey-sevastynov** [vercel/community#7381] [Account Issue] [answered]
### Question

Доброго дня, не можу зайти в сайт  з пулу ір 93.170.44.0/22
(Ukraine, Dnipro, SOYUZTELECOM) . У мене під час спроби відкриття  web-сторінки кабінету виникає помилка: "No connection, time it out". Скоріш за все встановлене  обмеження для цих IP адрес у Вас, або у Вагого хостинг провайдеру. Прошу скасувати це обмеження. При звернені до провайдера, провайдер повідовим, що блокування з їх сторони немає. 
(Hello, I can't access the site from the pool of IP 93.170.44.0/22 (Ukraine, Dnipro, SOYUZTELECOM). I get an error when I try to open the web page of my account: "No connection, time it out". Most likely, you or your hosting provider has set a restriction for these IP addresses. Please cancel this restriction. When you contact the provider, the provider will inform you that there is no blocking on their part.)

**Comment by pawlean** [comment, marked as answer]
Hi @alexey-sevastynov!

To help us debug further, could you use our [Vercel Connect script](https://github.com/vercel-support/vercel-connect-debug) and report back with results?

Thank you!

**Comment by alexey-sevastynov** [comment]
https://vercel.com/ - It's already working for me
https://drive.google.com/file/d/1hX9cVKLdYWDsNUVatQ2QHUMeVr8zL9B-/view?usp=sharing

пн, 8 июл. 2024 г. в 15:40, Pauline P. Narvas ***@***.***>:

**Reply by pawlean** [reply]
Hi @alexey-sevastynov

Do you mean it's working now?

**Comment by alexey-sevastynov** [comment]
Yes, it works! I log into the vecel platform without problems

вт, 9 июл. 2024 г. в 13:27, Pauline P. Narvas ***@***.***>:

[a comment by an automated account is not shown]
